### PR TITLE
[Stream] WebRTC beta docs

### PR DESCRIPTION
**Pull request by irvinebroque**

### CODE REVIEW

**Review by renandincer — APPROVED** [review]

File: content/stream/webrtc-beta.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+}
+```
+
+## Step 2: Go live using WHIP
+
+Every live input has a unique URL that one creator can be stream to. This URL should *only* be shared with the creator — anyone with this URL has the ability to stream live video to this live input.
+
+Copy the URL from the `webRTC` key in the API response (see above), or directly from the [Cloudflare Dashboard](https://dash.cloudflare.com/?to=/:account/stream/inputs).
+
+Paste this URL into the provided [WHIP example code](https://github.com/cloudflare/templates/tree/main/stream/webrtc), which you can [run in your web browser on Stackblitz](https://workers.new/stream/webrtc-whip):
```
Comment (dcpena) [inline content/stream/webrtc-beta.md]: Is this WHIP example code supposed to link to the top level of the repo or one of the files in the `src` folder?

File: content/stream/webrtc-beta.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+{{<Aside>}}
+[Trickle ICE](https://datatracker.ietf.org/doc/rfc8838/) is not yet supported, but will be supported soon. Some WHIP clients require Trickle ICE.
+{{</Aside>}}
+
+## Step 3: Play live video using WHEP
+
+Copy the URL from the `webRTCPlayback` key in the API response (see above), or directly from the [Cloudflare Dashboard](https://dash.cloudflare.com/?to=/:account/stream/inputs). There are no limits on the number of concurrent viewers.
+
+Paste this URL into the provided WHEP example code, which you [can run in your browser on Stackblitz]((https://workers.new/stream/webrtc-whep)).
```
Comment (dcpena) [inline content/stream/webrtc-beta.md]: Two sets of parentheses here which is causing a page not found.

File: content/stream/webrtc-beta.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+## Debugging WebRTC
+
+- **Chrome**: Navigate to `chrome://webrtc-internals` tov view detailed logs and graphs.
+- **Firefox**: Navigate to `about:webrtc` to view information about WebRTC sessions, similar to Chrome.
+- **Safari**: To enable WebRTC logs, from the inspector, open the settings tab (cogwheel icon), and set WebRTC logging to "Verbose" in the dropdown menu.
+
+## Limitations while in beta
+
+- [Recording](/stream/stream-live/replay-recordings/) is not yet supported (coming soon)
```
Comment (dcpena) [inline content/stream/webrtc-beta.md]: This is a broken link

File: content/stream/webrtc-beta.md (hunk 2 above)
Comment (irvinebroque) [reply content/stream/webrtc-beta.md]: Derp, thanks

File: content/stream/webrtc-beta.md (hunk 1 above)
Comment (irvinebroque) [inline content/stream/webrtc-beta.md]:
```suggestion
Paste this URL into the provided [WHIP example code](https://github.com/cloudflare/templates/blob/main/stream/webrtc/src/whip.html), which you can [run in your web browser on Stackblitz](https://workers.new/stream/webrtc-whip):
```

File: content/stream/webrtc-beta.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+Paste this URL into the provided WHEP example code, which you [can run in your browser on Stackblitz]((https://workers.new/stream/webrtc-whep)).
+
+```javascript
+---
+header: Simplified example code
+highlight: [4]
+---
+// Add a <video> element to the HTML page this code runs in:
+// <video id="input-video" autoplay muted></video>
```
Comment (Everlag) [inline content/stream/webrtc-beta.md]: swap id to be `output-video`? That way if they're doing this all on one page no collision

File: content/stream/webrtc-beta.md (hunk 2 above)
Comment (irvinebroque) [inline content/stream/webrtc-beta.md]:
```suggestion
Paste this URL into the provided WHEP example code, which you [can run in your browser on Stackblitz](https://workers.new/stream/webrtc-whep).
```

File: content/stream/webrtc-beta.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+import WHEPClient from "./WHEPClient.js"; // an example WHEP client, see https://github.com/cloudflare/templates/tree/main/stream/webrtc/src/WHEPClient.ts
+
+const url = "<WEBRTC_URL_FROM_YOUR_LIVE_INPUT>"; // add the webRTCPlayback URL from your live input here
+const videoElement = document.getElementById("input-video");
```
Comment (Everlag) [inline content/stream/webrtc-beta.md]: same query about id

File: content/stream/webrtc-beta.md (hunk 3 above)
Comment (irvinebroque) [inline content/stream/webrtc-beta.md]:
```suggestion
- [Recording](/stream/stream-live/replay-recordings/) is not yet supported (coming soon)
```

File: content/stream/webrtc-beta.md (hunk 1 above)
Comment (irvinebroque) [inline content/stream/webrtc-beta.md]:
```suggestion
Paste this URL into the provided [WHIP example code](https://github.com/cloudflare/templates/blob/main/stream/webrtc/src/whip.html#L13), which you can [run in your web browser on Stackblitz](https://workers.new/stream/webrtc-whip):
```

File: content/stream/webrtc-beta.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+const client = new WHEPClient(url, videoElement);
+```
+
+As long as the creator is actively streaming, viewers should see their broadcast in their browser, with less than 1 second of latency.
+
+You can also use this URL with any client that supports the [WebRTC-HTTP egress protocol (WHEP)](https://www.ietf.org/id/draft-murillo-whep-00.html).
```
Comment (Everlag) [inline content/stream/webrtc-beta.md]: Maybe directly mention we are working on broadening compatibility with different WHEP clients and our goal is to be fully compliant with others in the ecosystem?

File: content/stream/webrtc-beta.md (hunk 3 above)
Comment (dcpena) [reply content/stream/webrtc-beta.md]: I think this is the path we want. 

/stream/stream-live/watch-live-stream/#replaying-recordings

File: content/stream/webrtc-beta.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+- **Firefox**: Navigate to `about:webrtc` to view information about WebRTC sessions, similar to Chrome.
+- **Safari**: To enable WebRTC logs, from the inspector, open the settings tab (cogwheel icon), and set WebRTC logging to "Verbose" in the dropdown menu.
+
+## Limitations while in beta
+
+- [Recording](/stream/stream-live/replay-recordings/) is not yet supported (coming soon)
+- [Simulcasting](/stream/stream-live/simulcasting) (restreaming) is not yet supported (coming soon)
+- [Live viewer counts](/stream/getting-analytics/live-viewer-count/) are not yet supported (coming soon)
+- [Analytics](/stream/getting-analytics/fetching-bulk-analytics/) are not yet supported (coming soon)
+- Though we don't anticipate major API changes, while in beta, the WHIP and WHEP protocol versions used by our APIs is subject to change without notice. You can find the version in the `protocol-version` header in API responses. The value of this header references the IETF draft slug for each protocol, for example, `draft-ietf-wish-whip-04` and `draft-murillo-whep-00`.
```
Comment (Everlag) [inline content/stream/webrtc-beta.md]: Consider `API responses` -> `WHIP or WHEP responses' since its unclear which API this is referring to.

**Review by Everlag — APPROVED** [review]
Mostly LGTM, the id change is somewhat blocking

File: content/stream/webrtc-beta.md [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+
+Paste this URL into the provided WHEP example code, which you [can run in your browser on Stackblitz](https://workers.new/stream/webrtc-whep).
+
+```javascript
+---
+header: Simplified example code
+highlight: [4]
+---
+// Add a <video> element to the HTML page this code runs in:
+// <video id="input-video" autoplay muted></video>
```
Comment (irvinebroque) [inline content/stream/webrtc-beta.md]:
```suggestion
// <video id="output-video" autoplay muted></video>
```

File: content/stream/webrtc-beta.md [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+
+import WHEPClient from "./WHEPClient.js"; // an example WHEP client, see https://github.com/cloudflare/templates/tree/main/stream/webrtc/src/WHEPClient.ts
+
+const url = "<WEBRTC_URL_FROM_YOUR_LIVE_INPUT>"; // add the webRTCPlayback URL from your live input here
+const videoElement = document.getElementById("input-video");
```
Comment (irvinebroque) [inline content/stream/webrtc-beta.md]:
```suggestion
const videoElement = document.getElementById("output-video");
```

File: content/stream/webrtc-beta.md [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+const client = new WHEPClient(url, videoElement);
+```
+
+As long as the creator is actively streaming, viewers should see their broadcast in their browser, with less than 1 second of latency.
+
+You can also use this URL with any client that supports the [WebRTC-HTTP egress protocol (WHEP)](https://www.ietf.org/id/draft-murillo-whep-00.html).
```
Comment (irvinebroque) [inline content/stream/webrtc-beta.md]:
```suggestion
You can also use this URL with any client that supports the [WebRTC-HTTP egress protocol (WHEP)](https://www.ietf.org/id/draft-murillo-whep-00.html). As more WHEP clients are published, we are committed to supporting them and being fully compliant with the WHEP protocol as it evolves into a standard.
```

File: content/stream/webrtc-beta.md (hunk 3 above)
Comment (irvinebroque) [reply content/stream/webrtc-beta.md]: Discussing offline, something's weird here.

File: content/stream/webrtc-beta.md [hunk 11]
```diff
@@ -0,0 +1,125 @@
...
+
+## Debugging WebRTC
+
+- **Chrome**: Navigate to `chrome://webrtc-internals` tov view detailed logs and graphs.
+- **Firefox**: Navigate to `about:webrtc` to view information about WebRTC sessions, similar to Chrome.
+- **Safari**: To enable WebRTC logs, from the inspector, open the settings tab (cogwheel icon), and set WebRTC logging to "Verbose" in the dropdown menu.
+
+## Limitations while in beta
+
+- [Recording](/stream/stream-live/replay-recordings/) is not yet supported (coming soon)
```
Comment (irvinebroque) [inline content/stream/webrtc-beta.md]:
```suggestion
- [Recording](/stream/stream-live/watch-live-stream/#replaying-recordings) is not yet supported (coming soon)
```

File: content/stream/webrtc-beta.md [hunk 12]
```diff
@@ -0,0 +1,125 @@
...
+- [Simulcasting](/stream/stream-live/simulcasting) (restreaming) is not yet supported (coming soon)
+- [Live viewer counts](/stream/getting-analytics/live-viewer-count/) are not yet supported (coming soon)
+- [Analytics](/stream/getting-analytics/fetching-bulk-analytics/) are not yet supported (coming soon)
+- Though we don't anticipate major API changes, while in beta, the WHIP and WHEP protocol versions used by our APIs is subject to change without notice. You can find the version in the `protocol-version` header in API responses. The value of this header references the IETF draft slug for each protocol, for example, `draft-ietf-wish-whip-04` and `draft-murillo-whep-00`.
```
Comment (irvinebroque) [inline content/stream/webrtc-beta.md]:
```suggestion
- Though we don't anticipate major API changes, while in beta, the WHIP and WHEP protocol versions used by our APIs is subject to change without notice. You can find the version in the `protocol-version` header in WHIP and WHEP API responses. The value of this header references the IETF draft slug for each protocol, for example, `draft-ietf-wish-whip-04` and `draft-murillo-whep-00`.
```

**Pull request merged**